docs/devel/README.md

```diff
@@ -6,7 +6,7 @@ Before you being you must have:
 * [GitHub account][github]
 * `git`
 * `go`
-* `Docker`
+* `Docker` | `Podman`
```

Comment:
`s/|/or/`

Reply:
IMO, better to have "a container runtime" here and have a separate section that lists the supported options.